### PR TITLE
add Tiden logo

### PR DESCRIPTION
**Pull request by popoaq**
created new logo for Tiden, update logo.svg and favicon.ico

image

tiden_logo

### CODE REVIEW

File: frontend/yarn.lock [hunk 1]
```diff
@@ -2516,7 +2516,7 @@
   "resolved" "https://registry.npmjs.org/asap/-/asap-2.0.6.tgz"
   "version" "2.0.6"
 
-"asn1.js@^5.2.0":
+"asn1.js@^5.0.0", "asn1.js@^5.2.0":
```
Comment (jw122) [inline frontend/yarn.lock:2519]: nit: do these need to be checked in? same with package-lock
I've had to manually remove them from git locally, maybe .gitignore isn't picking up the stuff in frontend/

**Review by jw122 — APPROVED** [review]
beautiful

File: frontend/yarn.lock (hunk 1 above)
Comment (popoaq) [reply frontend/yarn.lock:2519]: yes, it looks like it should be checked in 
image
https://classic.yarnpkg.com/en/docs/yarn-lock/
image
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Comment (jw122) [reply frontend/yarn.lock:2519]: ah I see, thanks for the info! I need to start doing this

**Pull request merged**